code/game/gamemodes/mixed/paranormal.dm

```diff
@@ -0,0 +1,11 @@
+/datum/game_mode/paranormal
+	name = "Paranormal (Cult + Vamp)"
```

Comment:
This style is inconsistent with the other gamemodes - there should be no spacing around the `+`.